ios/ConverseNotificationExtension/Spam.swift

```diff
@@ -185,7 +185,7 @@ func computeSpamScoreV3Message(client: XMTP.Client, conversation: XMTP.Conversat
 
       if let senderAddresses = try await group.members.first(where: {$0.inboxId == senderInboxId})?.addresses {
         for address in senderAddresses {
-          if try await client.preferences.addressState(address:address) == .denied {
+          if try await client.preferences.addressState(address: address) == .denied {
```

Comment:
🤷 dunno how this got changed

Reply:
Yeah this will need a rebase, would say to not acccept anything unless you specifically know it should have changed

Reply:
let's rebase together then - early on here that will save time so i don't fuck something up simple

Reply:
new diff is just formatted correctly, so leaving